Makefile

```diff
@@ -211,9 +211,12 @@ else ifeq ($(CHAIN),songbird)
 APP_LOAD_PARAMS += --path "44'/554'" --path "44'/60'"
 DEFINES += CHAINID_UPCASE=\"SONGBIRD\" CHAINID_COINNAME=\"SGB\" CHAIN_KIND=CHAIN_KIND_SONGBIRD CHAIN_ID=19
 APPNAME = "Songbird"
-else
+else ifeq ($(CHAIN),shyft)
+APP_LOAD_PARAMS += --path "44'/7341'"
+DEFINES += CHAINID_UPCASE=\"SHYFT\" CHAINID_COINNAME=\"SHFT\" CHAIN_KIND=CHAIN_KIND_SHYFT CHAIN_ID=7341
+APPNAME = "Shyft"
```

Comment:
missing `else` at the end

Reply:
can I ask why you removed the `else` at the end? :)

Reply:
mistake! done